proto/controlclient.proto

```diff
@@ -55,3 +56,17 @@ message _RevokeSigningKeyRequest {
 
 message _RevokeSigningKeyResponse {
 }
+
+message _SigningKey {
+  string key_id = 1;
+  uint64 expires_at = 2;
```

Comment:
Nit: I fully realize nothing else in here is commented. Would you consider putting comments on the fields in these new messages?

Reply:
Sure thing!